code/game/objects/noose.dm

```diff
@@ -0,0 +1,153 @@
+/obj/item/stack/cable_coil/building_checks(datum/stack_recipe/R, multiplier)
+	if(R.title == "noose")
+		if(!(locate(/obj/structure/chair) in usr.loc) && !(locate(/obj/structure/bed) in usr.loc) && !(locate(/obj/structure/table) in usr.loc) && !(locate(/obj/structure/toilet) in usr.loc))
```

Comment:
Bad.

Reply:
`		if(!(locate(/obj/structure/chair) || (/obj/structure/table) || (/obj/structure/toilet) in usr.loc))`
Better?

Reply:
`		if(!(locate(/obj/structure/chair in usr.loc)))`
bit better?

Reply:
```
/obj/item/stack/cable_coil/building_checks(datum/stack_recipe/R, multiplier)
	if(R.title == "noose")
		if(!(locate(/obj/structure/chair in usr.loc)))
			to_chat(usr, "<span class='warning'>You have to be standing on top of a chair to make a noose!</span>")
			return FALSE
	return ..()
```

Reply:
```
/obj/item/stack/cable_coil/building_checks(datum/stack_recipe/R, multiplier)
	if(R.title == "noose")
		if(!(locate(/obj/structure/chair) in usr.loc))
			to_chat(usr, "<span class='warning'>You have to be standing on top of a chair to make a noose!</span>")
			return FALSE
	return ..()

```
bit better?

Reply:
toliet noose